test/metabase/api/query_description_test.clj

```diff
@@ -31,51 +31,61 @@
                                                          {:filter [:= [$price 1234]]}))))))
 
       (testing "with order-by clause"
-        (is (= {:table "Venues"
+        (is (= {:table    "Venues"
                 :order-by [{:field "Price" :direction :asc}]}
                (sut/generate-query-description (Table (mt/id :venues))
                                                (:query (mt/mbql-query :venues
                                                          {:order-by [[:asc $price]]}))))))
 
       (testing "with an aggregation metric"
-        (tt/with-temp Metric [metric {:table_id (mt/id :venues) :name "Test Metric 1"
+        (tt/with-temp Metric [metric {:table_id   (mt/id :venues) :name "Test Metric 1"
                                       :definition {:aggregation [[:count]]}}]
-          (is (= {:table "Venues"
+          (is (= {:table       "Venues"
                   :aggregation [{:type :metric
-                                 :arg "Test Metric 1"}]}
+                                 :arg  "Test Metric 1"}]}
                  (sut/generate-query-description (Table (mt/id :venues))
                                                  (:query (mt/mbql-query :venues
-                                                          {:aggregation [[:metric (:id metric)]]}))))))
+                                                           {:aggregation [[:metric (:id metric)]]}))))))
 
-        (is (= {:table "Venues"
+        (is (= {:table       "Venues"
                 :aggregation [{:type :metric
-                               :arg (deferred-tru "[Unknown Metric]")}]}
+                               :arg  (deferred-tru "[Unknown Metric]")}]}
                (sut/generate-query-description (Table (mt/id :venues))
                                                (:query (mt/mbql-query :venues
                                                          {:aggregation [[:metric -1]]}))))))
 
       (testing "with segment filters"
         (tt/with-temp Segment [segment {:name "Test Segment 1"}]
-          (is (= {:table "Venues"
+          (is (= {:table  "Venues"
                   :filter [{:segment "Test Segment 1"}]}
                  (sut/generate-query-description (Table (mt/id :venues))
                                                  (:query (mt/mbql-query :venues
                                                            {:filter [[:segment (:id segment)]]}))))))
 
-        (is (= {:table "Venues"
+        (is (= {:table  "Venues"
                 :filter [{:segment (deferred-tru "[Unknown Segment]")}]}
                (sut/generate-query-description (Table (mt/id :venues))
                                                (:query (mt/mbql-query :venues
-                                                         {:filter [[:segment -1]]})))))))))
+                                                         {:filter [[:segment -1]]}))))))
+
+      (testing "with named aggregation"
```

Comment:
It would be nice to also have a test for what broke: an unnamed metric referencing multiple fields.